pom.xml

```diff
@@ -90,10 +90,6 @@
     </developer>
   </developers>
 
-  <prerequisites>
```

Comment:
Prerequisites are not actually enforced in Maven 3, see: http://stackoverflow.com/a/18058572/33791